assembly/assembly-ide-war/pom.xml

```diff
@@ -347,6 +345,19 @@
                 <artifactId>exec-maven-plugin</artifactId>
                 <executions>
                     <execution>
+                        <id>gwt-xml</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>java</goal>
+                        </goals>
+                        <configuration>
+                            <mainClass>org.eclipse.che.util.CompilingGwtXmlGenerator</mainClass>
```

Comment:
I think I've got your idea, and I'm kind of agree with it. But I would like to delay this optimization.